go/oasis-test-runner/scenario/e2e/runtime/early_query.go

```diff
@@ -10,31 +10,39 @@ import (
 	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/env"
 	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/oasis"
 	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/scenario"
+	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/scenario/e2e"
 )
 
 var (
 	// EarlyQuery is the early query scenario where we query a validator node before the network
 	// has started and there are no committed blocks.
 	EarlyQuery scenario.Scenario = &earlyQueryImpl{
-		Scenario: *NewScenario("early-query"),
+		Scenario: e2e.NewScenario("early-query"),
 	}
 
 	// EarlyQueryInitHeight is the same as EarlyQuery scenario but with an initial height set.
 	EarlyQueryInitHeight scenario.Scenario = &earlyQueryImpl{
-		Scenario:      *NewScenario("early-query/init-height"),
+		Scenario:      e2e.NewScenario("early-query/init-height"),
 		initialHeight: 42,
 	}
+
+	EarlyQueryRuntime scenario.Scenario = &earlyQueryImpl{
```

Comment:
Add documentation comment.

Reply:
Check again please, I also changed so that in the runtime scenario all queries are done against the compute node (to not re-test querying the validator as it is already done in other tests)